cmp/store/impl/StoreFilterFieldImplModel.js

```diff
@@ -203,4 +202,27 @@ export class StoreFilterFieldImplModel extends HoistModel {
         }
         return ret;
     }
+
+
+    getValGetters(field) {
+        const {gridModel} = this,
+            cols = gridModel ? filter(gridModel.getVisibleLeafColumns(), {field}) : null;
+
+        // No associated visible column -- use simple get.
+        if (!cols) {
+            // lodash get() slower - use only when needed to support dot-separated paths.
+            return field.includes('.') ? (rec) => get(rec.data, field) : (rec) => rec.data[field];
+        }
+
+        // For each visible column, return closest approximation of rendered value.
+        return cols.map(column => {
+            const {renderer, getValueFn} = column;
+            return (record) => {
+                let ret = getValueFn({record, field, column, gridModel});
```

Comment:
We should call with the `store` as well - that's part of our API:
https://github.com/xh/hoist-react/blob/c13147124f2fbcaf05f39bb0e7680a9bfe06304b/cmp/grid/columns/Column.js#L714

Reply:
I suppose the weirdest thing that would happen with stripping commas is that you would get *more* matches than you wanted to on text when you were looking for a specific comma (e.g. in a company name) -- but that seems pretty unlikely.  Lets do that.